electrum_nmc/electrum/gui/qt/main_window.py

```diff
@@ -3341,7 +3341,10 @@ def create_buy_names_tab(self):
         # TODO: allow hex names
         self.buy_names_new_name_lineedit = self.buy_names_ui.registerName
         self.buy_names_new_name_lineedit.textChanged.connect(self.update_buy_names_preview)
```

Comment:
Please rename these to have `ascii` in the variable names, like you did for hex below.